app/assets/javascripts/discourse/widgets/notification-item.js.es6

```diff
@@ -148,6 +148,7 @@ createWidget("notification-item", {
     let infoKey = notName === "custom" ? data.message : notName;
     let text = emojiUnescape(this.text(notificationType, notName));
     let icon = iconNode(`notification.${infoKey}`);
+    let title = I18n.t(`notifications.titles.${notName}`);
```

Comment:
I think it might be nice to handle the case when the key is not present. Could we use an empty string as the default perhaps?